ui/src/components/harvest-games/HarvestGamesPage.js

```diff
@@ -401,10 +425,13 @@ const HarvestGames = (props) => {
                     </Link>
                     .
                   </Text>
-                </TabPanel>
-                <TabPanel height={{ base: '80vh', sm: '90vh' }}>
+                </TabPanel> */}
+                <TabPanel minHeight={{ base: '30em', sm: '40em' }}>
```

Comment:
not sure why you changed this since this actually breaks the harvest games leaderboard